react/Static.tsx

```diff
@@ -0,0 +1,5 @@
+import { injectIntl } from 'react-intl'
+
+import { RichText } from './index'
+
+export default injectIntl(RichText)
```

Comment:
I would add a comment here to explain what is happening. It took me a while to understand how is this different from the usual rich-text.

Reply:
I just wanna remove the schema, so it won't be available in to edit on CMS. It's a poor solution to  an uncommon situation

Reply:
I agree with @klzns , add a comment on the code here explaining what this does and why this was done. I'm not sure I fully understand to be honest